language-tour.md

```diff
@@ -241,6 +288,9 @@ public fun main() {
   print(weight) // 2.3
 }
 ```
+{% comment %}
+The ability to use (e.g.) an integer literal to initialize a double is IIUC one of the great sources of expression too complex errors in Swift.
+{% endcomment %}
```

Comment:
My beliefs about Swift's implementation come from this [blog post](https://www.cocoawithlove.com/blog/2016/07/12/type-checker-issues.html) (which might not be completely up to date). I remember working on that problem for the language I developed in my thesis and managing to infer relatively "complex" arithmetic expressions just by bounding the set of types that may conform to `ExpressibleBy***Literal`.

But I never formally investigated the complexity of my inference algorithm, neither of Swift's. So it's fair not to avoid making strong statements at this point.